### PR TITLE
Storage Write API: Adds conversion for Bytes schema type

### PR DESCRIPTION
**Pull request by b-goyal**
This PR adds logic to support Bytes in BigQuery table schema. Storage Api support ByteString for a bytes type. (You may find the related google support ticket in below mentioned JIRA ticket)

This PR contains additional logic to covert map to JSONObject as new JSONObject(map) does not support complex types. 
ByteString was getting converted to {empty: false} JSONObject due to JSOnObject's internal implementation. 

After making these changes, records following below schema has been tested with both old and new API.
Screenshot 2023-05-09 at 12 27 29 PM
Screenshot 2023-05-09 at 12 27 49 PM

JIRA: CC-20287
EPIC: CC-19644

### CODE REVIEW

**Review by pbadani — APPROVED** [review]
LGTM.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java [hunk 1]
```diff
@@ -121,5 +123,26 @@ public Runnable build() {
             }
             return new StorageWriteApiWriter(tableName, streamWriter, records, streamName);
         }
+
+        private JSONObject getJsonFromMap(Map<String, Object> map) {
+            JSONObject jsonObject = new JSONObject();
+            map.forEach((key, value) -> {
+                if (value instanceof Map<?, ?>) {
+                    value = getJsonFromMap((Map<String, Object>) value);
```
Comment (sp-gupta) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java:131]: @b-goyal Is it possible that the **value** here is an instance of anything but **Map<String, Object>**. In that case, I think the typecast won't work.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java [hunk 2]
```diff
@@ -121,5 +123,26 @@ public Runnable build() {
...
+                } else if (value instanceof List<?>) {
+                    JSONArray items = new JSONArray();
+                    ((List<?>) value).forEach(v -> {
+                        if(v instanceof Map<?,?>) {
+                            items.put(getJsonFromMap((Map<String, Object>) v));
```
Comment (sp-gupta) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java:136]: Same here

Comment (sp-gupta) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java:136]: Same here

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java (hunk 1 above)
Comment (b-goyal) [reply kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiWriter.java:131]: No @sp-gupta , that should not be possible. You may take a look at the checks and map conversion logic in BigQueryRecordConverter. First that class performs conversion and then we receive the map object here.

**Pull request merged**